tools/pipelines/templates/include-set-package-version.yml

```diff
@@ -100,6 +102,8 @@ steps:
     targetType: 'inline'
     workingDirectory: ${{ parameters.buildDirectory }}
     script: |
+      # Note: deliberately not using `set -eux -o pipefail` because this script leverages the return code of grep
+      # even in an error case
```

Comment:
We can replace this with a command in ~10 minutes. Let me see if I can whip something up. Not blocking this change.

Reply:
Not in a rush to get this in either, it's more of an experiment but if we get things green without too much effort, I'd definitely like to merge it when we have a chance.